app/Resources/views/themes/ezplatform/pagelayout.html.twig

```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
-<html class="no-js">
+<html class="no-js"
+      lang="en">
```

Comment:
I think you can improve your PR using W3C recommendations
```
<html lang="en" xml:lang="en" xmlns= "http://www.w3.org/1999/xhtml">
```
Also, if you can keep everything on the same line, it would be better. Consistency across the templates. :)

Reply:
Single-line updated, and meta tag introduced